### ISSUE TITLE
[RLC-410W] Streams become unavailable when used

### ISSUE THREAD

**Issue by dhzl84** [xannor/ha_reolink_rest#17]
Hello, just migrated from the "Reolink IP camera" integration.

When newly set up, three streams are available but with same quality.
After short time of usage the streams become unavailable and the following message is written to the logs.

Reloading the integration brings back the streams instantly, also waiting for some time seems to recover the streams.

```
Logger: homeassistant.components.websocket_api.http.connection
Source: custom_components/reolink_rest/camera.py:402
Integration: Home Assistant WebSocket API (documentation, issues)
First occurred: 15:35:30 (1 occurrences)
Last logged: 15:35:30

[140268327742256] Error handling message: Unknown error (unknown_error)
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/websocket_api/decorators.py", line 27, in _handle_async_response
    await func(hass, connection, msg)
  File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 805, in ws_camera_stream
    url = await _async_stream_endpoint_url(hass, camera, fmt=msg["format"])
  File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 953, in _async_stream_endpoint_url
    stream = await camera.async_create_stream()
  File "/usr/src/homeassistant/homeassistant/components/camera/__init__.py", line 524, in async_create_stream
    source = await self.stream_source()
  File "/config/custom_components/reolink_rest/camera.py", line 402, in stream_source
    url = await client.get_rtmp_url(
  File "/usr/local/lib/python3.10/site-packages/async_reolink/api/network/__init__.py", line 172, in get_rtmp_url
    await self._ensure_ports_and_link()
  File "/usr/local/lib/python3.10/site-packages/async_reolink/api/network/__init__.py", line 124, in _ensure_ports_and_link
    self.__link = response.local_link
AttributeError: 'GetNetworkPortsResponse' object has no attribute 'local_link'
```

<img width="591" alt="image" src="https://user-images.githubusercontent.com/5675570/190652815-d7b2f66d-f201-42b9-b6b3-84f09edf194c.png">

If you need more logging I can support.
I can also test a RLC-510W.

**Comment by xannor** [comment]
I found this issue and fixed in the latest release 0.6.3 (66106b0) give it a try and let me know.

**Comment by dhzl84** [comment]
Tested with 0.6.6 and no issues anymore.
The different streams (Main, Extra, Sub) are now reliable shown in their defined quality.

Well done and thank you!
